js/challenge6/index.js

```diff
@@ -0,0 +1,31 @@
+// //Generating Unique Random Numbers
+let arrayOfNum = [];
+
+while (arrayOfNum.length < 15) {
+  let randomNum = Math.floor(Math.random() * 16);
+  if (!arrayOfNum.includes(randomNum)) {
+    arrayOfNum.push(randomNum);
+  }
+}
+console.log(arrayOfNum);
+
+//generateNumbers function
+let newArrayOfNum = [];
+
+const generateNumbers = (n, r, isUnique = false) => {
+  if (n > r || n === 0) return;
+
```

Comment:
Because you're adding 1 to the range of values in the code below (r + 1), you can afford to have n be one more than r (i.e. n > r + 1). Otherwise, great job!

Reply:
thank you